custom/icds_reports/ucr/reports/asr/asr_2_3_person_cases.json

```diff
@@ -262,6 +262,18 @@
         "whens": {
           "closed_on IS NULL AND disability_type ~ '\\y5\\y'": 1
         }
+      },
+      {
+        "sortable": false,
+        "description": null,
+        "format": "default",
+        "aggregation": "count_unique",
+        "column_id": "awc_count",
+        "field": "awc_id",
+        "transform": {},
+        "calculate_total": true,
+        "type": "field",
+        "display": "awc_count"
```

Comment:
are these additions intentional? These are in a mobile restore correct? we generally don't add fields to mobile reports once they are deployed

Reply:
Yes, these are very much intentional. This is [needed](https://github.com/dimagi/commcare-hq/pull/24428/files#diff-add9ae2786c8fb702d2ea2d99e2e56a3R9) to avoid Python agg. 

Isn't there a flag to turn not display these in the mobile UCR?

Reply:
Its possible that these reports are displayed in a way that this would not show up to the user, but I'm not sure how easy it is to verify that.

The safe way to update these reports is:

1. copy this to a new report
2. change the report_id in the new report
3. change the report that is referenced in block_mpr.json and block_asr.json

Reply:
I've added a new source (don't like that we will be breaking DRY, but haven't found a simpler solution to avoid)